mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -0,0 +1,213 @@
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/coreos/mantle/platform/machine/unprivqemu"
+)
+
+var (
+	bootmirror = conf.Ignition(`{
+		"ignition": {
+			"version": "3.2.0"
+		},
+		"storage": {
+			"disks": [
+			  {
+				"device": "/dev/vda",
+				"partitions": [
+				  {
+					"label": "bios-1",
+					"sizeMiB": 1,
+					"typeGuid": "21686148-6449-6E6F-744E-656564454649"
+				  },
+				  {
+					"label": "esp-1",
+					"sizeMiB": 127,
+					"typeGuid": "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
+				  },
+				  {
+					"label": "boot-1",
+					"sizeMiB": 384
+				  },
+				  {
+					"label": "root-1"
+				  }
+				],
+				"wipeTable": true
+			  },
+			  {
+				"device": "/dev/vdb",
+				"partitions": [
+				  {
+					"label": "bios-2",
+					"sizeMiB": 1,
+					"typeGuid": "21686148-6449-6E6F-744E-656564454649"
+				  },
+				  {
+					"label": "esp-2",
+					"sizeMiB": 127,
+					"typeGuid": "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
+				  },
+				  {
+					"label": "boot-2",
+					"sizeMiB": 384
+				  },
+				  {
+					"label": "root-2"
+				  }
+				],
+				"wipeTable": true
+			  },
+			  {
+				"device": "/dev/vdc",
+				"partitions": [
+				  {
+					"label": "bios-3",
+					"sizeMiB": 1,
+					"typeGuid": "21686148-6449-6E6F-744E-656564454649"
+				  },
+				  {
+					"label": "esp-3",
+					"sizeMiB": 127,
+					"typeGuid": "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
+				  },
+				  {
+					"label": "boot-3",
+					"sizeMiB": 384
+				  },
+				  {
+					"label": "root-3"
+				  }
+				],
+				"wipeTable": true
+			  }
+			],
+			"filesystems": [
+			 { 
+				"device": "/dev/disk/by-partlabel/esp-1",
+				"format": "vfat",
+				"label": "esp-1",
+				"wipeFilesystem": true
+			  },
+			  {
+				"device": "/dev/disk/by-partlabel/esp-2",
+				"format": "vfat",
+				"label": "esp-2",
+				"wipeFilesystem": true
+			  },
+			  {
+				"device": "/dev/disk/by-partlabel/esp-3",
+				"format": "vfat",
+				"label": "esp-3",
+				"wipeFilesystem": true
+			  },
+			  {
+				"device": "/dev/md/md-boot",
+				"format": "ext4",
+				"label": "boot",
+				"wipeFilesystem": true
+			  },
+			  {
+				"device": "/dev/md/md-root",
+				"format": "xfs",
+				"label": "root",
+				"wipeFilesystem": true
+			  }
+			],
+			"raid": [
+			  {
+				"devices": [
+				  "/dev/disk/by-partlabel/boot-1",
+				  "/dev/disk/by-partlabel/boot-2",
+				  "/dev/disk/by-partlabel/boot-3"
+				],
+				"level": "raid1",
+				"name": "md-boot",
+				"options": [
+				  "--metadata=1.0"
+				]
+			  },
+			  {
+				"devices": [
+				  "/dev/disk/by-partlabel/root-1",
+				  "/dev/disk/by-partlabel/root-2",
+				  "/dev/disk/by-partlabel/root-3"
+				],
+				"level": "raid1",
+				"name": "md-root"
+			  }
+			]
+		  }
+	}`)
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Run:         runBootMirrorTest,
+		ClusterSize: 0,
+		Name:        `fcos.ignition.boot-mirror`,
```

Comment:
This should probably be `coreos.` not `fcos.`, I don't see anything inherently preventing this test from being ran on RHCOS & the test itself isn't currently restricting the distro to `fcos` only.